api/meetings/serializers.py

```diff
@@ -149,25 +145,8 @@ def get_download_count(self, obj):
         if getattr(obj, 'download_count', None):
             return obj.download_count or 0
         else:
-            node_ct = ContentType.objects.get_for_model(AbstractNode).id
-            with connection.cursor() as cursor:
-                cursor.execute(
-                    """
-                    SELECT P.total
-                    FROM osf_basefilenode F, osf_pagecounter P
-                    WHERE (F.type = 'osf.osfstoragefile'
-                         AND F.provider = 'osfstorage'
-                         AND F.target_content_type_id = %s
-                         AND F.target_object_id = %s
-                         AND P._id = 'download:' || %s || ':' || F._id)
-                    ORDER BY F.id ASC
-                    LIMIT 1;
-                """, [node_ct, obj.id, obj._id],
-                )
-                result = cursor.fetchone()
-                if result:
-                    return int(result[0])
-                return 0
+            submission_file = self.get_submission_file(obj)
+            return submission_file.get_download_count() if submission_file else None
```

Comment:
This query was only being run on MeetingSubmissionDetail endpoint previously - annotated count was used on List endpoint. Current implementation now uses this for List and Detail.  Calling get_download_count() on just ten files brings this to acceptable speeds.
[Attempted to optimize SQL here ](https://github.com/CenterForOpenScience/osf.io/pull/9061/commits/06c430d5a32d92fede58fb70ca91b58c16ddc3e5) but still too slow with PageCounter model issues.